tests/Microsoft.Identity.Web.Test/L1L2CacheTests.cs

```diff
@@ -42,38 +42,114 @@ public async Task WriteCache_WritesInL1L2_TestAsync()
             byte[] cache = new byte[3];
             AssertCacheValues(_testCacheAdapter);
             Assert.Equal(0, _testCacheAdapter._memoryCache.Count);
-            Assert.Empty(L2Cache.dict);
+            Assert.Empty(L2Cache._dict);
 
             // Act
             await _testCacheAdapter.TestWriteCacheBytesAsync(DefaultCacheKey, cache).ConfigureAwait(false);
 
             // Assert
             Assert.Equal(1, _testCacheAdapter._memoryCache.Count);
-            Assert.Single(L2Cache.dict);
+            Assert.Single(L2Cache._dict);
         }
 
         [Fact]
         public async Task WriteCache_NegativeExpiry_TestAsync()
```

Comment:
MSAL should never ask you to persist something with negative expiry, this would be a bug. Not sure what the best behavior is in this case: 

- throw an exception
- write the value to the cache, which semantically means "DELETE key" and log an error. If this happens often, it affects the cache hit ratio.

Thoughts?